data-raw/vs_peds.R

```diff
@@ -183,11 +241,7 @@ vs_peds <- tibble::tribble(
     VSEVAL = NA_character_,
     EPOCH = "Epoch"
   ) %>%
-  select(
-    STUDYID, DOMAIN, USUBJID, VSSEQ, VSTESTCD, VSTEST,
-    VSORRES, VSORRESU, VSSTRESC, VSSTRESN, VSSTRESU, VSEVAL,
-    EPOCH, VISIT, VISITNUM, VSDTC, VSDY
-  )
+  arrange(USUBJID, VISITNUM, VSTESTCD, VSDY)
 
 # get common column names with VS
```

Comment:
this time this block does seem to be needed as `dplyr::bind_rows()` clears the labels but when i run this block it doesn't seem to keep the labels. maybe use `metatools::add_labels()` to set the variable labels instead like in for example https://github.com/pharmaverse/pharmaversesdtm/blob/main/data-raw/ce_vaccine.R

Reply:
The issue has been resolved by updating the current code for setting the labels for variables comm with vs and separately adding labels for two newly added variables, VSEVAL and EPOCH.